components/ui/tabs-variants.tsx

```
@@ -0,0 +1,99 @@
+);
+
+export interface TabsListProps
+  extends React.ComponentPropsWithoutRef<typeof TabsPrimitive.List>,
+    VariantProps<typeof TabsListVariants> {
+  asChild?: boolean;
+}
+
+const TabsList = React.forwardRef<
+  React.ElementRef<typeof TabsPrimitive.List>,
+  TabsListProps
+>(({ className, variant, width, asChild = false, ...props }, ref) => (
```

ESLint

[error] 'asChild' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)